libs/s25main/ingameWindows/iwMilitaryBuilding.cpp

```diff
@@ -32,6 +32,13 @@ iwMilitaryBuilding::iwMilitaryBuilding(GameWorldView& gwv, GameCommandFactory& g
                    _(BUILDING_NAMES[building->GetBuildingType()]), LOADER.GetImageN("resource", 41)),
       gwv(gwv), gcFactory(gcFactory), building(building)
 {
+    unsigned offset = 0;
```

Comment:
Maybe a better name: 
```suggestion
    unsigned btOffset = 0;
```

so that it is clear it refers to the buttons, especially as it is declared a bit away from its use